src/app/modules/verification/verification.controller.ts

```diff
@@ -53,6 +54,45 @@ export const handleCreateTransaction: RequestHandler<
     })
 }
 
+/**
+ * NOTE: Private handler for POST /forms/:formId/fieldverifications
+ * When a form is loaded publicly, a transaction is created, and populated with the field ids of fields that are verifiable.
+ * If no fields are verifiable, then it did not create a transaction and returns an empty object.
+ * @param req
+ * @param res
+ * @returns 201 - transaction is created
+ * @returns 200 - transaction was not created as no fields were verifiable for the form
+ */
+export const handleCreateTransactionWithFieldId: RequestHandler<
```

Comment:
the handler name does not seem to match its intended behaviour? should it be `handleCreateVerificationTransaction` instead?